Converts obsolete use of defined typecast NULL pointers with NULL.

diff -Naur ./gnofract4d-3.13/fract4d/c/imageIO.cpp ./gnofract4d-3.13-new/fract4d/c/imageIO.cpp
--- ./gnofract4d-3.13/fract4d/c/imageIO.cpp	2010-03-28 04:47:10.000000000 +0200
+++ ./gnofract4d-3.13-new/fract4d/c/imageIO.cpp	2010-12-25 23:32:23.688361939 +0100
@@ -119,7 +119,7 @@
 	info_ptr = png_create_info_struct(png_ptr);
 	if(NULL == info_ptr)
 	{
-	    png_destroy_write_struct(&png_ptr, png_infopp_NULL);
+	    png_destroy_write_struct(&png_ptr, NULL);
 	    return;
 	}
 
@@ -312,7 +312,7 @@
 	info_ptr = png_create_info_struct(png_ptr);
 	if (info_ptr == NULL)
 	{
-	    png_destroy_read_struct(&png_ptr, png_infopp_NULL, png_infopp_NULL);
+	    png_destroy_read_struct(&png_ptr, NULL, NULL);
 	    return;
 	}
 
@@ -325,7 +325,7 @@
 	//printf("shutdown\n");
 	if(ok)
 	{
-	    png_destroy_read_struct(&png_ptr, &info_ptr, png_infopp_NULL);
+	    png_destroy_read_struct(&png_ptr, &info_ptr, NULL);
 	}
     }
 
@@ -351,7 +351,7 @@
     
     //printf("get IHDR\n");
     png_get_IHDR(png_ptr, info_ptr, &width, &height, &bit_depth, &color_type,
-		 &interlace_type, int_p_NULL, int_p_NULL);
+		 &interlace_type, NULL, NULL);
 
     //printf("set res(%d,%d)\n",width,height);
     if(!im->set_resolution(width, height, -1, -1))
@@ -372,7 +372,7 @@
 	for (int y = 0; y < im->Yres(); y++)
 	{
 	    png_bytep row = (png_bytep)(im->getBuffer() + im->row_length() * y); 
-	    png_read_rows(png_ptr, &row, png_bytepp_NULL, 1);
+	    png_read_rows(png_ptr, &row, NULL, 1);
 	}
     }
     return true;
